### ISSUE TITLE
pytorch yolov5

### ISSUE THREAD

**Issue by mrfan98** [Mazin-Hnewa/MS-DAYOLO#5]
When I used yolov5 model to train cityScapes' 2975 training sets to test the cityScapes - Foggy validation set, I was always told that no labels were found

**Comment by Mazin-Hnewa** [comment]
For yolov5, you need to put images and labels in different folders, while here in this repo, we use YOLOv4 which images and labels should be in the same folder.

**Comment by mrfan98** [comment]
> For yolov5, you need to put images and labels in different folders, while here in this repo, we use YOLOv4 which images and labels should be in the same folder.

What does target set (with dummy annotation) mean when you training,  it is have a label or not

**Comment by Mazin-Hnewa** [comment]
Normally for domain adaptation, labels are not available for target data.
The target set does not have any labels, but we put dummy annotations (i.e. empty files) to load target data with yolov4.

**Comment by mrfan98** [comment]
> Normally for domain adaptation, labels are not available for target data. The target set does not have any labels, but we put dummy annotations (i.e. empty files) to load target data with yolov4.

ok, thanks
